controllers/admin/AdminModulesController.php

```diff
@@ -145,43 +143,6 @@ public function setMedia($isNewTheme = false)
         }
     }
 
-    public function ajaxProcessRefreshModuleList($force_reload_cache = false)
-    {
-        // Refresh default country native modules list every day
-        if (!Tools::isFileFresh(Module::CACHE_FILE_DEFAULT_COUNTRY_MODULES_LIST, 86400) || $force_reload_cache) {
```

Comment:
customer modules OK
must-have modules OK
But why country modules ?

Reply:
country modules are native modules ?

Reply:
Because this code is written from `Tools::addonsRequest('native')` which was removed.